docs/user/security/reporting.asciidoc

```diff
@@ -9,19 +9,56 @@ To use {reporting} with {security} enabled, you need to
 <<using-kibana-with-security,set up {kib} to work with {security}>>.
 If you are automatically generating reports with
 {ref}/xpack-alerting.html[{watcher}], you also need to configure {watcher}
-to trust the {kib} server's certificate. For more information, see
+to trust the {kib} server's certificate. 
+////
+For more information, see
 <<securing-reporting>>.
+////
 
 [[reporting-app-users]]
-To enable users to generate reports, assign them the built-in `reporting_user`
+To enable users to generate reports, you must assign them the built-in `reporting_user`
 role.  Users will also need the appropriate <<kibana-privileges, {kib} privileges>> to access the objects 
 to report on and the {es} indices.
 
-* If you're using the `native` realm, you can assign roles through
-**Management > Users** UI in Kibana or with the `user` API. For example,
-the following request creates a `reporter` user that has the
-`reporting_user` role and the `kibana_user` role:
+
+==== If you are using the `native` realm
+
+You can assign roles through the
+*Management* app in Kibana or with the {ref}/security-api-put-user.html[user API]. 
+This example shows how to use *Management* to create a user who has a custom role and the 
+`reporting_user` role.
+
+. Go to *Management > Roles*, and click *Create role*.
+  
+. Give the new role a name, for example, `custom_reporting_user`.
+
+. Specify the indices and privileges.
 +
+Access to data is an index-level privilege, so in *Create role*, 
+add a line for each index that contains the data for the report and give each 
+index `read` and `view_index_metadata` privileges.
+For more information, see {ref}/security-privileges.html[Security privileges]. 
++
+[role="screenshot"]
+image::user/security/images/reporting-privileges-example.png["Reporting privileges"]
+
+. Add space privileges.
++
+Reporting users typically save searches, create 
+visualizations, and build dashboards. They require a space 
+that provides read and write privileges in
+*Discover*, *Visualize*, and *Dashboard*.
+
+. Save your new role.
+
+. Create a user account with the proper roles.
++
+Go to *Management > Users*, add a new user, and assign the user the built-in 
+`reporting_user` role and your new custom role, `custom_reporting_user`.
+
+This example uses the `user` API to create a user who has the
```

Comment:
This section on creating a native realm user with the API seems like it should have a header. 

Can the above link with the `user API` text link to this section?

Perhaps the outward-directing link that goes to `security-api-put-user` could be moved down to here?